src/runtest.rs

```diff
@@ -211,8 +211,12 @@ impl<'test> TestCx<'test> {
 
     fn check_correct_failure_status(&self, proc_res: &ProcRes) {
         // The value the rust runtime returns on failure
-        const RUST_ERR: i32 = 1;
-        if proc_res.status.code() != Some(RUST_ERR) {
+        let rust_err: i32 = if cfg!(feature = "stable") {
```

Comment:
I was afraid of this. The whole "support stable Rust" is starting to become unmaintainable :-/ The `check_correct_failure_status()` function is somewhat different in rustc/compiletest so this fix isn't that back-portable.

I suppose it's fine for now, but I think a bigger sync-up with the current rustc/compiletest would be a good idea. I just worry that it would stable support.

Would it be better to skip the tests for the stable feature?

Reply:
Yeah, I agree

Reply:
OK, thanks, I've also filed #121. Once this passes, we should be good to go. Thanks again!